src/crypto/secp256k1.ts

```diff
@@ -1,20 +1,27 @@
 import ecc from "secp256k1";
-import { randomBytes } from "crypto";
+import {randomBytes} from "crypto";
+import {Bip39, EnglishMnemonic, Slip10, Slip10Curve, Slip10RawIndex} from "@cosmjs/crypto"
 
 export class Secp256k1 {
-  static generatePrivateKey(): Uint8Array {
-    let privKey
-    do {
-      privKey = randomBytes(32)
-    } while (!ecc.privateKeyVerify(privKey));
-    return privKey;
-  }
+    static generatePrivateKey(): Uint8Array {
```

Comment:
As Hansol said, please use 2 spaces for indentations.

Reply:
Yes, I fixed it.
Intellij applied 4 spaces by default.